docs/OMF.rst

```diff
@@ -235,6 +235,7 @@ Formats & Types
 ~~~~~~~~~~~~~~~
 
 The *Formats & Types* tab provides a means to specify the detail types that will be used and the way complex assets are mapped to OMF types to also be configured.
+See the section *Numeric Data Types* below for more information on configuring data types.
```

Comment:
I think we should really make these links, see the LinkedTypes for an example.

Reply:
reference link issue fixed in PR-https://github.com/fledge-iot/fledge/pull/1302